### ISSUE TITLE
Can not install after I have uninstalled

### ISSUE THREAD

**Issue by ghost** [prometheus-community/windows_exporter#511]
Hello there, first of all great project I use this on around 100 servers and it's so useful for metrics and alerts!

I've had this issues a few times where it consumes all of the servers available memory on version 0.10.2 (sorry I haven't reported this bug actually) so to get round this I just uninstall it and then install an older version 0.9.0 

However this time when I'm running the MSI it's telling me "A later version of WMI Exporter is already installed. Setup will now exit." 

There is no service running, there is no install directory in Program Files, also looked through the registry and deleted anything I could find related to wmi_exporter but it is still giving me the same error.

So in short, I can not install it on this server again which is extremely frustrating.

**Comment by marvtec** [comment]
Also when trying to install 0.10.2 that I have previously uninstalled I get the following error from windows installer;

**The configuration data for this product is corrupt.
Contact you support personnel.**

**Comment by bsnuggs1** [comment]
Hey @marvtec, there's another issue (#208) that references how to stop and uninstall the program.  I don't have an answer for your config data being corrupted though, sorry.

**Comment by carlpett** [comment]
@marvtec It would be really helpful if you could open an issue for the memory leak as well, but let's look at your uninstall problems first.
In some situations (eg pending install of Windows Updates or some files being locked during uninstallation, but probably others as well) the Windows Installer subsystem will not finalize the uninstallation without a reboot. Would it be possible for you to see if a reboot clears this up?

**Comment by marvtec** [comment]
Hi @carlpett  I had already tried rebooting before posting, this unfortunately did not solve the issue.

**Comment by sergkkaltura** [comment]
Try to add "REINSTALL=ALL" parameter, solved that for me . should look like: 
msiexec /i <path-to-msi-file> ENABLED_COLLECTORS=os,iis LISTEN_PORT=5000 REINSTALL=ALL

**Comment by kang2681** [comment]
Try to delete  HKEY_CLASSES_ROOT\Installer\Products\5A52F83B614406F428F729D2B0F239A0

**Comment by birkanyaylaci** [comment]
I removed it from Control Panel\Programs\Programs and Features. It is solved

**Comment by sbudaraj-vcc** [comment]
> REINSTALL=ALL

this saves me a ton of time, thanks a lot!
